draft-ietf-rtcweb-jsep.xml

```diff
@@ -3163,6 +3164,935 @@ a=end-of-candidates
+a=candidate:4036177504 1 udp 1685987071 55.66.77.88 64532 typ srflx
+            raddr 192.168.2.3 rport 61665
+a=candidate:3671762467 1 udp 41819903 66.77.88.99 50416 typ relay
+            raddr 55.66.77.88 rport 64532
+a=end-of-candidates
+
+m=video 64532 UDP/TLS/RTP/SAVPF 100 101
+c=IN IP4 55.66.77.88
+a=rtcp:64532 IN IP4 55.66.77.88
+a=mid:v1
+a=msid:61317484-2ed4-49d7-9eb7-1414322a7aae
+       f30bdb4a-5db8-49b5-bcdc-e0c9a23172e0
+a=sendrecv
+a=rtpmap:100 VP8/90000
+a=rtpmap:101 rtx/90000
+a=fmtp:101 apt=100
+a=ice-ufrag:7sFvz2gdLkEwjZEr
+a=ice-pwd:dOTZKZNVlO9RSGsEGM63JXT2
+a=ice-options:trickle
+a=fingerprint:sha-256
+              19:E2:1C:3B:4B:9F:81:E6:B8:5C:F4:A5:A8:D8:73:04
+             :BB:05:2F:70:9F:04:A9:0E:05:E9:26:33:E8:70:88:A2
+a=setup:actpass
+a=rtcp-mux
+a=rtcp-rsize
+a=extmap:2 urn:ietf:params:rtp-hdrext:sdes:mid
+a=rtcp-fb:100 ccm fir
+a=rtcp-fb:100 nack
+a=rtcp-fb:100 nack pli
+a=ssrc:1366781083 cname:Q/NWs1ao1HmN4Xa5
+a=ssrc:1366781084 cname:Q/NWs1ao1HmN4Xa5
+a=ssrc-group:FID 1366781083 1366781084
+a=candidate:109270924 1 udp 2122194687 192.168.2.3 61665 typ host
+a=candidate:4036177504 1 udp 1685987071 55.66.77.88 64532 typ srflx
+            raddr 192.168.2.3 rport 61665
+a=candidate:3671762467 1 udp 41819903 66.77.88.99 50416 typ relay
+            raddr 55.66.77.88 rport 64532
+a=end-of-candidates
+
+m=video 64532 UDP/TLS/RTP/SAVPF 100 101
+c=IN IP4 55.66.77.88
+a=rtcp:64532 IN IP4 55.66.77.88
+a=mid:v1
+a=msid:71317484-2ed4-49d7-9eb7-1414322a7aae
+       f30bdb4a-5db8-49b5-bcdc-e0c9a23172e0
+a=sendrecv
+a=rtpmap:100 VP8/90000
+a=rtpmap:101 rtx/90000
+a=fmtp:101 apt=100
+a=ice-ufrag:7sFvz2gdLkEwjZEr
+a=ice-pwd:dOTZKZNVlO9RSGsEGM63JXT2
+a=ice-options:trickle
+a=fingerprint:sha-256
+              19:E2:1C:3B:4B:9F:81:E6:B8:5C:F4:A5:A8:D8:73:04
+             :BB:05:2F:70:9F:04:A9:0E:05:E9:26:33:E8:70:88:A2
+a=setup:actpass
+a=rtcp-mux
+a=rtcp-rsize
+a=extmap:2 urn:ietf:params:rtp-hdrext:sdes:mid
+a=rtcp-fb:100 ccm fir
+a=rtcp-fb:100 nack
+a=rtcp-fb:100 nack pli
+a=ssrc:2366781083 cname:Q/NWs1ao1HmN4Xa5
+a=ssrc:2366781084 cname:Q/NWs1ao1HmN4Xa5
+a=ssrc-group:FID 2366781083 2366781084
+a=candidate:109270924 1 udp 2122194687 192.168.2.3 61665 typ host
+a=candidate:4036177504 1 udp 1685987071 55.66.77.88 64532 typ srflx
+            raddr 192.168.2.3 rport 61665
+a=candidate:3671762467 1 udp 41819903 66.77.88.99 50416 typ relay
+            raddr 55.66.77.88 rport 64532
+a=end-of-candidates
+
+         ]]></artwork>
+       </figure>
+     </t>
+
+     <t>
+       The SDP for |answer-C1| which looks like:
+     </t>
+       <t>
+       <figure>
+         <artwork><![CDATA[
+v=0
+o=- 4962303333179871723 1 IN IP4 0.0.0.0
+s=-
+t=0 0
+a=msid-semantic:WMS
+a=group:BUNDLE a1 d1 v1 v2
+m=audio 52546 UDP/TLS/RTP/SAVPF 96 0 8 97 98
+c=IN IP4 11.22.33.44
+a=rtcp:52546 IN IP4 11.22.33.44
+a=mid:a1
+a=msid:57017fee-b6c1-4162-929c-a25110252400
+       e83006c5-a0ff-4e0a-9ed9-d3e6747be7d9
+a=inactvie
```

Comment:
typo on inactive. This should probably be the same SDP as the previous complex example, just with a=inactive.